README.md

```diff
@@ -128,14 +128,14 @@ Please read through our [contributing guidelines](https://github.com/plotly/plot
 
 * Follow [@plotlygraphs](https://twitter.com/plotlygraphs) on Twitter for the latest Plotly news.
 * Follow [@plotly_js](https://twitter.com/plotly_js) on Twitter for plotly.js release updates.
-* Implementation help may be found on community.plot.ly (tagged [`plotly-js`](http://community.plot.ly/c/plotly-js)) or
+* Implementation help may be found on community.plot.com (tagged [`plotly-js`](https://community.plotly.com/c/plotly-js)) or
   on Stack Overflow (tagged [`plotly`](https://stackoverflow.com/questions/tagged/plotly)).
 * Developers should use the keyword `plotly` on packages which modify or add to the functionality of plotly.js when distributing through [npm](https://www.npmjs.com/browse/keyword/plotly).
-* Direct developer email support can be purchased through a [Plotly Support Plan](https://support.plot.ly/libraries/javascript).
+* Direct developer email support can be purchased through a [Plotly Support Plan](https://plotly.com/chart-studio-help/chart-studio-cloud-support/).
```

Comment:
Please remove this line.

Reply:
Done in https://github.com/plotly/plotly.js/pull/4694/commits/1fe9c53d66973721d0a512c4570b9c7d612b1f90.